website/pages/index.tsx

```diff
@@ -72,6 +72,7 @@ const previewWrapper = css`
   &:hover {
     & > div {
       opacity: 1;
+      transform: translate3d(0, -${spacing[3]}px, 0) scale(0.95);
```

Comment:
We should always make the transitioned state of an element be the default transform values: `translate3d(0, 0, 0) scale(1)`.

Reply:
Got it, fixed!